{"uid":"678a60919658a41e","name":"test_when_there_is_no_video_playout_a_waiting_video[chromium]","fullName":"tests.Automation_tests.test_admin_playout#test_when_there_is_no_video_playout_a_waiting_video","historyId":"2abd14c0f6b30f6060a16d10dadb2089","time":{"start":1758895245241,"stop":1758895245241,"duration":0},"status":"broken","statusMessage":"requests.exceptions.ConnectionError: HTTPConnectionPool(host='10.189.200.6', port=80): Max retries exceeded with url: /api/v1/authenticate (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11678f9d0>: Failed to establish a new connection: [Errno 51] Network is unreachable'))","statusTrace":"E   OSError: [Errno 51] Network is unreachable\n\nThe above exception was the direct cause of the following exception:\nE   urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x11678f9d0>: Failed to establish a new connection: [Errno 51] Network is unreachable\n\nThe above exception was the direct cause of the following exception:\nE   urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='10.189.200.6', port=80): Max retries exceeded with url: /api/v1/authenticate (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11678f9d0>: Failed to establish a new connection: [Errno 51] Network is unreachable'))\n\nDuring handling of the above exception, another exception occurred:\nE   requests.exceptions.ConnectionError: HTTPConnectionPool(host='10.189.200.6', port=80): Max retries exceeded with url: /api/v1/authenticate (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11678f9d0>: Failed to establish a new connection: [Errno 51] Network is unreachable'))","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"test_setup","time":{"start":1758892666166,"stop":1758892674974,"duration":8808},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false,"stepsCount":0,"hasContent":false},{"name":"base_url","time":{"start":1758892666166,"stop":1758892666166,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false,"stepsCount":0,"hasContent":false},{"name":"pytestconfig","time":{"start":1758892666166,"stop":1758892666166,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false,"stepsCount":0,"hasContent":false},{"name":"delete_output_dir","time":{"start":1758892666166,"stop":1758892666166,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false,"stepsCount":0,"hasContent":false},{"name":"check_if_secrets_are_loaded","time":{"start":1758892666166,"stop":1758892666166,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false,"stepsCount":0,"hasContent":false},{"name":"_verify_url","time":{"start":1758892666166,"stop":1758892666166,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false,"stepsCount":0,"hasContent":false},{"name":"rp_logger","time":{"start":1758892674974,"stop":1758892674974,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false,"stepsCount":0,"hasContent":false},{"name":"playwright","time":{"start":1758892674975,"stop":1758892675126,"duration":151},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false,"stepsCount":0,"hasContent":false},{"name":"browser_type_launch_args","time":{"start":1758892674975,"stop":1758892674975,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false,"stepsCount":0,"hasContent":false},{"name":"browser","time":{"start":1758892675127,"stop":1758892675553,"duration":426},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false,"stepsCount":0,"hasContent":false},{"name":"launch_browser","time":{"start":1758892675127,"stop":1758892675127,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false,"stepsCount":0,"hasContent":false},{"name":"browser_type","time":{"start":1758892675127,"stop":1758892675127,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false,"stepsCount":0,"hasContent":false},{"name":"admin_user","time":{"start":1758895245241,"stop":1758895245242,"duration":1},"status":"broken","statusMessage":"requests.exceptions.ConnectionError: HTTPConnectionPool(host='10.189.200.6', port=80): Max retries exceeded with url: /api/v1/authenticate (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11678f9d0>: Failed to establish a new connection: [Errno 51] Network is unreachable'))\n","statusTrace":"  File \"/Users/s.chethana/PycharmProjects/wnec_backend-automation/.venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/s.chethana/PycharmProjects/wnec_backend-automation/.venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/s.chethana/PycharmProjects/wnec_backend-automation/.venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/s.chethana/PycharmProjects/Data_Driven_Framework_WNEC/conftest.py\", line 125, in fixture_admin_user\n    tokenized_user = API(api_util.auto_admin_user[\"username\"], api_util.auto_admin_user[\"password\"])\n                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/s.chethana/PycharmProjects/Data_Driven_Framework_WNEC/helpers/api.py\", line 20, in __init__\n    self.token = get_access_token(username, password)\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/s.chethana/PycharmProjects/Data_Driven_Framework_WNEC/helpers/api.py\", line 12, in get_access_token\n    token_request = requests.post(f\"{API_URL}/authenticate\", json=auth, verify=False, timeout=10)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/s.chethana/PycharmProjects/wnec_backend-automation/.venv/lib/python3.11/site-packages/requests/api.py\", line 115, in post\n    return request(\"post\", url, data=data, json=json, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/s.chethana/PycharmProjects/wnec_backend-automation/.venv/lib/python3.11/site-packages/requests/api.py\", line 59, in request\n    return session.request(method=method, url=url, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/s.chethana/PycharmProjects/wnec_backend-automation/.venv/lib/python3.11/site-packages/requests/sessions.py\", line 589, in request\n    resp = self.send(prep, **send_kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/s.chethana/PycharmProjects/wnec_backend-automation/.venv/lib/python3.11/site-packages/requests/sessions.py\", line 703, in send\n    r = adapter.send(request, **kwargs)\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/s.chethana/PycharmProjects/wnec_backend-automation/.venv/lib/python3.11/site-packages/requests/adapters.py\", line 700, in send\n    raise ConnectionError(e, request=request)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":0,"attachmentStep":false,"stepsCount":0,"hasContent":true}],"afterStages":[{"name":"browser::0","time":{"start":1758895245261,"stop":1758895245321,"duration":60},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false,"stepsCount":0,"hasContent":false},{"name":"playwright::0","time":{"start":1758895245322,"stop":1758895245324,"duration":2},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"attachmentStep":false,"stepsCount":0,"hasContent":false},{"name":"test_setup::0","time":{"start":1758895245325,"stop":1758895245327,"duration":2},"status":"broken","statusMessage":"requests.exceptions.ConnectionError: HTTPConnectionPool(host='10.189.200.6', port=80): Max retries exceeded with url: /api/v1/authenticate (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11378e190>: Failed to establish a new connection: [Errno 51] Network is unreachable'))\n","statusTrace":"  File \"/Users/s.chethana/PycharmProjects/wnec_backend-automation/.venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 231, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/s.chethana/PycharmProjects/wnec_backend-automation/.venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 911, in _teardown_yield_fixture\n    next(it)\n  File \"/Users/s.chethana/PycharmProjects/Data_Driven_Framework_WNEC/conftest.py\", line 101, in fixture_test_setup\n    FileDistributor().remove_all_disribution_process()\n  File \"/Users/s.chethana/PycharmProjects/Data_Driven_Framework_WNEC/component_util/file_distributor.py\", line 74, in remove_all_disribution_process\n    response, status = API().get(\"/distributionprocesses\")\n                       ^^^^^\n  File \"/Users/s.chethana/PycharmProjects/Data_Driven_Framework_WNEC/helpers/api.py\", line 20, in __init__\n    self.token = get_access_token(username, password)\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/s.chethana/PycharmProjects/Data_Driven_Framework_WNEC/helpers/api.py\", line 12, in get_access_token\n    token_request = requests.post(f\"{API_URL}/authenticate\", json=auth, verify=False, timeout=10)\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/s.chethana/PycharmProjects/wnec_backend-automation/.venv/lib/python3.11/site-packages/requests/api.py\", line 115, in post\n    return request(\"post\", url, data=data, json=json, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/s.chethana/PycharmProjects/wnec_backend-automation/.venv/lib/python3.11/site-packages/requests/api.py\", line 59, in request\n    return session.request(method=method, url=url, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/s.chethana/PycharmProjects/wnec_backend-automation/.venv/lib/python3.11/site-packages/requests/sessions.py\", line 589, in request\n    resp = self.send(prep, **send_kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/s.chethana/PycharmProjects/wnec_backend-automation/.venv/lib/python3.11/site-packages/requests/sessions.py\", line 703, in send\n    r = adapter.send(request, **kwargs)\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/s.chethana/PycharmProjects/wnec_backend-automation/.venv/lib/python3.11/site-packages/requests/adapters.py\", line 700, in send\n    raise ConnectionError(e, request=request)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":0,"attachmentStep":false,"stepsCount":0,"hasContent":true}],"labels":[{"name":"feature","value":"AdminPlayout"},{"name":"parentSuite","value":"tests.Automation_tests"},{"name":"suite","value":"test_admin_playout"},{"name":"host","value":"TR-GKJF6LPM4P"},{"name":"thread","value":"77029-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.Automation_tests.test_admin_playout"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"browser_name","value":"'chromium'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[]}],"history":{"statistic":{"failed":0,"broken":16,"skipped":0,"passed":4,"unknown":0,"total":20},"items":[{"uid":"148fc884d46e6ef3","status":"passed","time":{"start":1758805958179,"stop":1758805996937,"duration":38758}},{"uid":"b3158fe656734906","status":"passed","time":{"start":1758710790333,"stop":1758710816016,"duration":25683}},{"uid":"a91ef1981842dab8","status":"broken","statusDetails":"playwright._impl._errors.TimeoutError: Locator.click: Timeout 30000ms exceeded.\nCall log:\nwaiting for locator(\"//dialog[@class=\\\"inv-bg\\\"]//div//div[@class=\\\"row buttons\\\"]//button[2]\")","time":{"start":1758547986020,"stop":1758548040916,"duration":54896}},{"uid":"14e6134620293043","status":"broken","statusDetails":"playwright._impl._errors.TimeoutError: Locator.click: Timeout 30000ms exceeded.\nCall log:\nwaiting for locator(\"//dialog[@class=\\\"inv-bg\\\"]//div//div[@class=\\\"row buttons\\\"]//button[2]\")","time":{"start":1758539492840,"stop":1758539541896,"duration":49056}},{"uid":"57c54a94bb79dcbb","status":"broken","statusDetails":"playwright._impl._errors.TimeoutError: Locator.click: Timeout 30000ms exceeded.\nCall log:\nwaiting for locator(\"//dialog[@class=\\\"inv-bg\\\"]//div//div[@class=\\\"row buttons\\\"]//button[2]\")","time":{"start":1758538286495,"stop":1758538335565,"duration":49070}},{"uid":"45f498aa26d61c05","status":"passed","time":{"start":1758537202761,"stop":1758537252506,"duration":49745}},{"uid":"31b6dea4326670d5","status":"broken","statusDetails":"playwright._impl._errors.TimeoutError: Locator.click: Timeout 30000ms exceeded.\nCall log:\nwaiting for locator(\"//dialog[@class=\\\"inv-bg\\\"]//div//div[@class=\\\"row buttons\\\"]//button[2]\")","time":{"start":1757326712224,"stop":1757326762069,"duration":49845}},{"uid":"ef0ae6eba6822406","status":"broken","statusDetails":"playwright._impl._errors.TimeoutError: Locator.click: Timeout 30000ms exceeded.\nCall log:\nwaiting for locator(\"//dialog[@class=\\\"inv-bg\\\"]//div//div[@class=\\\"row buttons\\\"]//button[2]\")","time":{"start":1757316162204,"stop":1757316208798,"duration":46594}},{"uid":"cd691aa8d4d398f9","status":"passed","time":{"start":1756109575858,"stop":1756109628893,"duration":53035}},{"uid":"60aa82645be88b7e","status":"broken","statusDetails":"playwright._impl._errors.TimeoutError: Locator.click: Timeout 30000ms exceeded.\nCall log:\nwaiting for locator(\"//dialog[@class=\\\"inv-bg\\\"]//div//div[@class=\\\"row buttons\\\"]//button[2]\")","time":{"start":1756102663682,"stop":1756102712508,"duration":48826}},{"uid":"a5070568381aa153","status":"broken","statusDetails":"playwright._impl._errors.TimeoutError: Locator.click: Timeout 30000ms exceeded.\nCall log:\nwaiting for locator(\"//dialog[@class=\\\"inv-bg\\\"]//div//div[@class=\\\"row buttons\\\"]//button[2]\")","time":{"start":1755861404227,"stop":1755861444600,"duration":40373}},{"uid":"69612b27d5447830","status":"broken","statusDetails":"playwright._impl._errors.TimeoutError: Locator.click: Timeout 30000ms exceeded.\nCall log:\nwaiting for locator(\"//dialog[@class=\\\"inv-bg\\\"]//div//div[@class=\\\"row buttons\\\"]//button[2]\")","time":{"start":1755849600166,"stop":1755849640670,"duration":40504}},{"uid":"6b3c3a315003f8e1","status":"broken","statusDetails":"playwright._impl._errors.TimeoutError: Locator.click: Timeout 30000ms exceeded.\nCall log:\nwaiting for locator(\"//dialog[@class=\\\"inv-bg\\\"]//div//div[@class=\\\"row buttons\\\"]//button[2]\")","time":{"start":1755770703859,"stop":1755770754702,"duration":50843}},{"uid":"19cc180dbbf3c8c7","status":"broken","statusDetails":"playwright._impl._errors.TimeoutError: Locator.click: Timeout 30000ms exceeded.\nCall log:\nwaiting for locator(\"//dialog[@class=\\\"inv-bg\\\"]//div//div[@class=\\\"row buttons\\\"]//button[2]\")","time":{"start":1754668124359,"stop":1754668173398,"duration":49039}},{"uid":"59e53796c4f0430c","status":"broken","statusDetails":"playwright._impl._errors.TimeoutError: Locator.click: Timeout 30000ms exceeded.\nCall log:\nwaiting for locator(\"//dialog[@class=\\\"inv-bg\\\"]//div//div[@class=\\\"row buttons\\\"]//button[2]\")","time":{"start":1754305971985,"stop":1754306026379,"duration":54394}},{"uid":"8132faadf9cbd650","status":"broken","statusDetails":"playwright._impl._errors.TimeoutError: Locator.click: Timeout 30000ms exceeded.\nCall log:\nwaiting for locator(\"//dialog[@class=\\\"inv-bg\\\"]//div//div[@class=\\\"row buttons\\\"]//button[2]\")","time":{"start":1754047017250,"stop":1754047057673,"duration":40423}},{"uid":"5d5428d313bcc7e1","status":"broken","statusDetails":"playwright._impl._errors.TimeoutError: Locator.click: Timeout 30000ms exceeded.\nCall log:\nwaiting for locator(\"//div[@class=\\\"list column\\\"]//div[1]//div[@data-name=\\\"source\\\"]//div//ul//li[5]\")","time":{"start":1754036664993,"stop":1754036700276,"duration":35283}},{"uid":"40dd3fa0e205dad2","status":"broken","statusDetails":"playwright._impl._errors.TimeoutError: Locator.click: Timeout 30000ms exceeded.\nCall log:\nwaiting for locator(\"//div[@class=\\\"list column\\\"]//div[1]//div[@data-name=\\\"source\\\"]//div//ul//li[5]\")","time":{"start":1754032945696,"stop":1754032980986,"duration":35290}},{"uid":"d4c93f78b3fec00e","status":"broken","statusDetails":"playwright._impl._errors.TimeoutError: Locator.click: Timeout 30000ms exceeded.\nCall log:\nwaiting for locator(\"//dialog[@class=\\\"inv-bg\\\"]//div//div[@class=\\\"row buttons\\\"]//button[2]\")","time":{"start":1749553093150,"stop":1749553142165,"duration":49015}}]},"tags":[]},"source":"678a60919658a41e.json","parameterValues":["'chromium'"]}